R/preprocess.R

```diff
@@ -471,14 +466,16 @@ enw_filter_reference_dates <- function(obs, earliest_date, include_days,
 #' @return A `data.frame` of observations filtered for the latest available data
 #' for each reference date.
 #'
-#' @inheritParams check_dates
+#' @param obs An `data.frame`; must have `report_date` and `reference_date`
```

Comment:
An or A?

Reply:
yep; and fixed the 466 line, which should be a data.table